sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/http/client/JdkHttpClient.java

```diff
@@ -44,4 +45,12 @@ public JdkHttpClient(Object httpClient, Set<String> restrictedHeaders, Duration
     public Response<BinaryData> send(HttpRequest request) {
         throw LOGGER.throwableAtError().log(ERROR_MESSAGE, UnsupportedOperationException::new);
     }
+
+    @Override
+    public CompletableFuture<Response<BinaryData>> sendAsync(HttpRequest request) {
+        CompletableFuture<Response<BinaryData>> failedFuture = new CompletableFuture<>();
+        failedFuture.completeExceptionally(LOGGER.throwableAtError()
+            .log(ERROR_MESSAGE, UnsupportedOperationException::new));
+        return failedFuture;
```

Comment:
Can we instead call the sync method and wrap with CF instead of throwing? Since this is the default client, should we support both sync and async?

Reply:
The multi-jar version for Java 12 implements this method properly